### PR TITLE
Replace dev.hbar with account id

### PR DESCRIPTION
**Pull request by aconstas**
Replace all references to getAccountIdFromDomain() with account id  string ('0.0.1005415'). Retain getAccountIdFromDomain() for potential future use.

### CODE REVIEW

File: frontend/src/services/domain-service.ts [hunk 1]
```diff
@@ -3,9 +3,10 @@ import { Resolver, ICache } from '@pwoosam/hashgraph-name-resolution-sdk';
 const domainResolver = new Resolver('hedera_main');
 domainResolver.init();
 
-export const getAccountIdFromDomain = async (domain: string): Promise<string | undefined> => {
-  return await domainResolver.resolveSLD(domain);
-}
+// Unused for now due to the state of .hbar domains
+// export const getAccountIdFromDomain = async (domain: string): Promise<string | undefined> => {
```
Comment (pwoosam) [inline frontend/src/services/domain-service.ts]: We should add this back since this functionality still works, we're just opting out of using dev.hbar for nft explorer

File: frontend/src/components/layout/Navbar.tsx [hunk 2]
```diff
@@ -50,10 +50,8 @@ export const Navbar = () => {
             } else if (type === IdType.ACCOUNT) {
               navigate(`/account/${searchQuery}`);
             } else if (type === IdType.DOMAIN) {
-              const accId = await getAccountIdFromDomain(searchQuery);
-              if (accId) {
-                navigate(`/account/${accId}`);
-              }
+              const accId = '0.0.1005415';
+              navigate(`/account/${accId}`);
```
Comment (pwoosam) [inline frontend/src/components/layout/Navbar.tsx]: We should undo the changes in this file since it breaks searching by domain and will result in all domains going to the dev account.

Other users will still be continuing to use domains :)

File: frontend/src/components/hashconnect/support-progress.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { Box, LinearProgress, Typography } from "@mui/material";
 import { useEffect, useMemo, useState } from "react";
 import { getThisMonthsIncome } from "../../api-clients/hedera-mirror-node-api-helper";
-import { getAccountIdFromDomain } from "../../services/domain-service";
+// import { getAccountIdFromDomain } from "../../services/domain-service";
```
Comment (pwoosam) [inline frontend/src/components/hashconnect/support-progress.tsx]: We can remove this line instead of commenting it out

**Review by pwoosam — CHANGES_REQUESTED** [review]

**Comment by aconstas** [comment]
Updated after reviewing comments!

**Review by pwoosam — APPROVED** [review]

**Comment by pwoosam** [comment]
Fixes #3
